File: ts/src/BluefinPayconexError.ts

import { Context } from './Context'


class BluefinPayconexError extends Error {

  isBluefinPayconexError = true

  sdk = 'BluefinPayconex'

  code: string
  ctx: Context

  constructor(code: string, msg: string, ctx: Context) {
    super(msg)
    this.code = code
    this.ctx = ctx
  }

}

export {
  BluefinPayconexError
}

File: dart/lib/BluefinPayconexError.dart
class BluefinPayconexError extends Error {
  final bool isBluefinPayconexError = true;

  final String sdk = 'BluefinPayconex';

  String code;
  String message;
  dynamic ctx;

  // Populated by makeError with the (cleaned) result and spec.
  dynamic result;
  dynamic spec;

  BluefinPayconexError(this.code, this.message, [this.ctx]);

  @override
  String toString() => 'BluefinPayconexError: ' + code + ': ' + message;
}
